_posts/2024-08-22-boost-vector-search-with-css.md

```diff
@@ -0,0 +1,574 @@
+      <td>15.9</td>
+      <td>17.6</td>
+      <td>49</td>
+      <td>53.37</td>
+      <td>0.97</td>
+    </tr>
+    <tr>
+      <td>4</td>
+      <td>28</td>
+      <td>33</td>
+      <td>41</td>
+      <td>86</td>
+      <td>50.12</td>
+      <td>0.97</td>
+    </tr>
+    <tr>
+      <td rowspan="2">NMSLIB</td>
+      <td rowspan="2">383</td>
+      <td>1</td>
+      <td>14</td>
+      <td>15</td>
+      <td>16</td>
+      <td>29</td>
+      <td>51.12</td>
+      <td>0.97</td>
+    </tr>
+    <tr>
+      <td>4</td>
+      <td>21</td>
+      <td>25</td>
+      <td>31</td>
+      <td>72</td>
+      <td>42.63</td>
+      <td>0.97</td>
+    </tr>
+    <tr>
+      <td rowspan="2">Faiss</td>
+      <td rowspan="2">381</td>
+      <td>1</td>
+      <td>14</td>
+      <td>15</td>
+      <td>17</td>
+      <td>30</td>
+      <td>41.1</td>
+      <td>0.97</td>
+    </tr>
+    <tr>
+      <td>4</td>
+      <td>23</td>
+      <td>28</td>
+      <td>37</td>
+      <td>77</td>
+      <td>47.19</td>
+      <td>0.97</td>
+    </tr>
+  </tbody>
+</table>
+
+#### Experiment 5: Concurrent search enabled, max slice count = 8
+
+<table border="1">
+  <thead>
+    <tr>
+      <th>k-NN engine</th>
+      <th>Segment count</th>
+      <th>Num search clients</th>
+      <th colspan="3">Service time (ms)</th>
+      <th>Max CPU %</th>
+      <th>% JVM heap used</th>
+      <th>Recall</th>
+    </tr>
+    <tr>
+      <th></th>
+      <th></th>
+      <th></th>
+      <th>p50</th>
+      <th>p90</th>
+      <th>p99</th>
+      <th></th>
+      <th></th>
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td rowspan="2">Lucene</td>
+      <td rowspan="2">381</td>
+      <td>1</td>
+      <td>14</td>
+      <td>16</td>
+      <td>18</td>
+      <td>43</td>
+      <td>45.37</td>
+      <td>0.97</td>
+    </tr>
+    <tr>
+      <td>4</td>
+      <td>28</td>
+      <td>34</td>
+      <td>43</td>
+      <td>87</td>
+      <td>48.79</td>
+      <td>0.97</td>
+    </tr>
+    <tr>
+      <td rowspan="2">NMSLIB</td>
+      <td rowspan="2">383</td>
+      <td>1</td>
+      <td>10</td>
+      <td>12</td>
+      <td>14</td>
+      <td>41</td>
+      <td>45.21</td>
+      <td>0.97</td>
+    </tr>
+    <tr>
+      <td>4</td>
+      <td>23</td>
+      <td>25</td>
+      <td>29</td>
+      <td>75</td>
+      <td>45.87</td>
+      <td>0.97</td>
+    </tr>
+    <tr>
+      <td rowspan="2">Faiss</td>
+      <td rowspan="2">381</td>
+      <td>1</td>
+      <td>15</td>
+      <td>16</td>
+      <td>17</td>
+      <td>44</td>
+      <td>48.68</td>
+      <td>0.97</td>
+    </tr>
+    <tr>
+      <td>4</td>
+      <td>23</td>
+      <td>26</td>
+      <td>32</td>
+      <td>79</td>
+      <td>47.19</td>
+      <td>0.97</td>
+    </tr>
+  </tbody>
+</table>
+
+### Results comparison 
+
+For simplicity, we'll focus on the p90 metric from our experiments with a single search client, because this metric captures the performance of long-running vector search queries.
```

Comment:
```suggestion
For simplicity, we'll focus on the p90 metric with a single search client because this metric captures the performance of long-running vector search queries.
```